kolibri/plugins/learn/assets/src/vue/card-grid/grid-item/index.vue

```diff
@@ -40,24 +40,40 @@
     background-color: $core-bg-light
     overflow: hidden
     border-radius: $radius
+    @media screen and (max-width: $portrait-breakpoint)
+      width: $horizontal-card-width
+      height: $horizontal-card-height
 
   .thumb-wrapper
     position: relative
     display: block
-    float: left
-    width: $thumb-width
-    height: $card-height
-    border-radius: $radius 0 0  $radius
+    width: 100%
+    height: $thumbnail-height
+    border-radius: $radius $radius 0 0
     overflow: hidden
+    @media screen and (max-width: $portrait-breakpoint)
+      float: left
+      width: $thumb-width
+      height: $horizontal-card-height
+      border-radius: $radius 0 0  $radius
 
   .text
     display: block
-    float: right
-    width: $card-width - $thumb-width
-    height: $card-height
+    display: -webkit-box
+    -webkit-line-clamp: 3
+    -webkit-box-orient: vertical
```

Comment:
I don't think this is a standard behavior we can rely on - I assume it doesn't work in all browsers? Best to avoid prefixed attributes or add comments to the style indicating why they're being used.